server/routers/analysis.py

```
@@ -271,3 +343,43 @@ def get_disaster_types(db: Session = Depends(get_db)):
             for disaster_type, count, avg_severity in types
         ],
     }
+
+
+@router.get("/filter-options")
+def get_filter_options(db: Session = Depends(get_db)):
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)